react/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+Object.defineProperty(exports, '__esModule', {
+  value: true,
+});
+
+var library = require('./dist/index.js');
+
+Object.keys(library).forEach(function (key) {
+  if (key === 'default' || key === '__esModule') {
+    return;
+  }
+  if (key in exports && exports[key] === library[key]) {
+    return;
+  }
+  Object.defineProperty(exports, key, {
+    enumerable: true,
+    get: function () {
+      return library[key];
+    },
+  });
+});
```

Comment:
Stavo cercando di capire se possiamo fare a meno di questo.

Peró non ho chiaro i passaggi:
- https://github.com/ionic-team/stencil-ds-output-targets#react
- https://github.com/ionic-team/stencil-ds-react-template

Reply:
Era stato messo per avere una shortcut ai dist di React. Finché era la versione con `export * from './index.js';` funzionava abbastanza, ma Jest non la supporta, e così effettivamente è abbastanza brutto.
Tre strade direi:
1. `'@zanichelli/albe-web-components/react'` --> `'@zanichelli/albe-web-components/react/dist'`
2. Provare a fare la dist da `react/src` dentro a `react`, ma non mi piace molto....
3. Usare la cartella react come output per Stencil e fare il transpiling typescript in place (via di mezzo tra 1 e 2)